bitcoin/src/pow.rs

```diff
@@ -723,22 +723,6 @@ impl<T: Into<u128>> From<T> for U256 {
     fn from(x: T) -> Self { U256(0, x.into()) }
 }
 
-/// Error from `TryFrom<signed type>` implementations, occurs when input is negative.
-#[derive(Debug, Clone, PartialEq, Eq)]
-#[non_exhaustive]
-pub struct TryFromError(i128);
-
-impl fmt::Display for TryFromError {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "attempt to create unsigned integer type from negative number: {}", self.0)
-    }
-}
-
-#[cfg(feature = "std")]
-impl std::error::Error for TryFromError {
-    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> { None }
-}
-
```

Comment:
How is this unused? Didn't we want to use it?

Reply:
To answer "how" -- it has no constructors and  deleting it does not break the codebase. So it definitely was unused.

It first appeared in #1197 two years ago, a PR with 200+ comments that obviously underwent a ton of iterations. I think it showed up as a rebasing artifact. Possibly we did intend to use it, but the evidence for this is long-buried.

Reply:
I mean deleting it might be the wrong solution. But I think it was meant for `TryFrom<i*>` which is indeed dubious.

Reply:
Whatever, let' delete it, we can add it later anyway.